### DISCUSSION TITLE
Not all Environments available in Send Request?

### DISCUSSION THREAD

**Discussion by pwillart** [postmanlabs/postman-flows#412] [Defect]
I created an elaborate flow and tested on my local environment. Now I am ready to test it on a different environment and I notice that they don't appear anymore in the dropdown. I am pretty sure they showed up earlier. 

![image](https://user-images.githubusercontent.com/61402842/197913390-e48acbe4-72e8-43c7-a0b3-f7d35d672705.png)

![image](https://user-images.githubusercontent.com/61402842/197913590-35ababc4-8762-46a0-9a1c-c7de932d2434.png)

Any ideas what might be causing this?

**Comment by pwillart** [comment]
Never mind... I figured it out. You first have to change it to 'No Environment' and then all the environments become available for selection again. Not very intuitive Postman team.....

**Reply by daniel-albert-art** [reply]
I think this is due to the text box working as some sort of filter.

You don't need to switch to "No Environment", just clear the text box to show all environments.
